src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/RestoreAzureRMSqlDatabase.cs

```diff
@@ -122,15 +122,18 @@ public class RestoreAzureRmSqlDatabase
         /// </summary>
         [Parameter(Mandatory = false,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The database edition to use for the restored database.")]
+            HelpMessage = "The database edition to use for the restored database." +
+            "New supported database edition: GeneralPurpose and BusinessCritical")]
         public DatabaseEdition Edition { get; set; }
 
         /// <summary>
         /// Gets or sets the SLO of the database to restore
         /// </summary>
         [Parameter(Mandatory = false,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The service level objective to use for the restored database.")]
+            HelpMessage = "The service level objective to use for the restored database." +
+            "Existing supported database ServiceObjectiveName examples: S3, P2, DW100." +
+            "New supported database ServiceObjectiveName example: GP_Gen4_2, BC_Gen4_2")]
```

Comment:
Instead reference the Capabilities cmdlet to say what ServiceObjectiveNames are valid.

Reply:
@akromm will update.